corehq/apps/receiverwrapper/views.py

```diff
@@ -80,18 +84,11 @@ def _process_form(request, domain, app_id, user_id, authenticated,
             except:
                 meta = {}
 
-            details = [
-                "domain:{}".format(domain),
-                "app_id:{}".format(app_id),
-                "user_id:{}".format(user_id),
-                "authenticated:{}".format(authenticated),
-                "form_meta:{}".format(meta),
-            ]
-            datadog_counter(MULTIMEDIA_SUBMISSION_ERROR_COUNT, tags=details)
-            notify_exception(request, "Received a submission with POST.keys()", details)
-            response = HttpResponseBadRequest(six.text_type(e))
-            _record_metrics(metric_tags, 'unknown', response)
-            return response
+            return _submission_error(
+                request, "Received a submission with POST.keys()",
```

Comment:
small change here: this message is returned in the response rather than `six.text_type(e)`